### DISCUSSION TITLE
how to pass -var-file for opentofu projects?

### DISCUSSION THREAD

**Discussion by fatih-gumus** [semaphoreui/semaphore#2260] [Q&A]
i am recently exploring semaphore ui and i hit to a problem. i have several opentofu projects and i am trying to pass -var-file but i couldn't so far. what are the possible solutions for this?

Thanks in advance.

**Comment by MaxTheMidget** [comment]
I'm having a similar issue where I'm trying to load an additional tvar file to my terraform plan. I get the following with the cli argument "-var-file="terraform.tfvars" -var-file="vmconfig.tfvars"" Did you get to the bottom of this?
 Error: Failed to read variables file

│ Given variables file "terraform.tfvars" does not exist.

│ Given variables file "vmconfig.tfvars" does not exist.
